pkg/vm/vm_core.go

```diff
@@ -22,17 +22,15 @@ type VirtualMachine struct {
 	RunContext     RunContext
 	CurrentStep    uint
 	Segments       memory.MemorySegmentManager
-	BuiltinRunners []builtins.BuiltinRunner
 	Trace          []TraceEntry
 	RelocatedTrace []RelocatedTraceEntry
 }
 
 func NewVirtualMachine() *VirtualMachine {
 	segments := memory.NewMemorySegmentManager()
-	builtin_runners := make([]builtins.BuiltinRunner, 0, 9) // There will be at most 9 builtins
 	trace := make([]TraceEntry, 0)
 	relocatedTrace := make([]RelocatedTraceEntry, 0)
-	return &VirtualMachine{Segments: segments, BuiltinRunners: builtin_runners, Trace: trace, RelocatedTrace: relocatedTrace}
+	return &VirtualMachine{Segments: segments, Trace: trace, RelocatedTrace: relocatedTrace}
```

Comment:
Dont remove

Reply:
It is actually a merge conflict it seems. i'll fix it